tests/test_crawler_orchestration.py

```
@@ -0,0 +1,61 @@
+    db.enqueue = AsyncMock(return_value=True)
+
+    crawler = OnionCrawler(db)
+    await crawler._process_discovered_onions(
+        ["exampleexample.onion"],
+        referrer="http://seed.onion/",
+        current_depth=1,
+    )
+
+    db.register_onion.assert_called_once()
+    db.enqueue.assert_called_once()
+    args, kwargs = db.enqueue.call_args
```

Ruff (0.15.5)

[warning] Unpacked variable `kwargs` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)